main.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+"""
+open all 3 files
+parse such that we have the log in a list
+count how many lines are in log, level=error count, level=info
+
+TIME FORMAT : 08:56:35.354
+
+top 5 most requested HTTP paths/endpoints
+"""
+
+class Logging:
+    def __init__(self):
+        self.log = []
+        self.date_time = []
+
+
+    def logs(self, file1, file2, file3):
```

Comment:
The prompt did say you don't have to generalize things, but honestly the difference here is negligible. If you're gonna accept parameters that are three things of exactly the same type and purpose, just accept a list.

```python
def logs(self, files):
```